readthedocs/templates/core/project_detail_right.html

```diff
@@ -91,6 +91,15 @@ <h3>{% trans "Project Privacy Level" %}</h3>
   </div>
 {% endblock %}
 
+{% block project-default-branch %}
+  <h3>{% trans "Project Default Branch" %}</h3>
```

Comment:
From #3554 

>  The front dashboard page version listing should note the default branch

I think this info should go on the list, not on the side bar.

Reply:
Oh Yes. I thought so. But I wasn't clear about the requirement. 
It is there on the list as shown in the picture.
Should I remove these lines then?

Reply:
👍 on default version noted in the table, not the sidebar.